spec/models/project_step_spec.rb

```diff
@@ -36,4 +36,20 @@
     expect(step[:original_date]).not_to be_nil
   end
 
+  context 'groups' do
```

Comment:
I try to do the same but I'm not religious about it. If I'm creating a bunch of factory objects in the before or lets then I tend to feel bad about having a ton of it blocks because that's a lot of extra database traffic which slows things down.